proto/buf.yaml

```diff
@@ -4,6 +4,7 @@
 # buf.yaml
 #
 version: v1
+name: buf.build/tendermint/fundraising
```

Comment:
@ilgooz do we have this Buf registry namespace?

Reply:
```suggestion
name: buf.build/ignitehq/fundraising
```